config/cluster-baremetal-operator/cluster-baremetal-operator.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cluster-baremetal-operator
+  name: operator
```

Comment:
Is this change correct?

Reply:
yes, see https://github.com/openshift/cluster-baremetal-operator/pull/87/files#diff-f557e464031f4695b338c76660478a714639a766096c14b2fc4ebb385afe11e2R2